### ISSUE TITLE
docker_compose_no_pip only works in Ansible 2.7 or later

### ISSUE THREAD

**Issue by cojack** [haxorof/ansible-role-docker-ce#68]
Hi,
I have encounter a problem with installation of docker-compose without pip:

```
 [WARNING]: - haxorof.docker-ce (1.10.0) is already installed - use --force to change version to unspecified

... lot of tasks ...

TASK [haxorof.docker-ce : Install docker-compose (Linux)] *****************************************************************************************
Thursday 29 November 2018  18:45:46 +0100 (0:00:02.852)       0:02:30.120 ***** 
fatal: [34.249.45.132]: FAILED! => {"changed": false, "msg": "The checksum parameter has to be in format <algorithm>:<checksum>"}
```

Any advice?

```
ansible --version
ansible 2.6.5
  config file = /home/cojack/Projects/demo1/ansible.cfg
  configured module search path = [u'/home/cojack/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15+ (default, Oct  2 2018, 22:12:08) [GCC 8.2.0]
```

**Comment by haxorof** [comment]
Hi,
Thanks for reporting this. Is it possible for you to run ansible-playbook with added argument -vvv and just paste the output for the failing task?

Cheers!

**Comment by haxorof** [comment]
Hi again @cojack!
In addition to what information I would like to receive from you, could you also provide which Linux distribution you are using and version. That will help me to see if there is a general problem related to that distribution and version.

Thanks!

**Comment by haxorof** [comment]
Regression tests has passed for all the distributions this role support but I found a bug which cause docker compose to be installed with pip even when `docker_compose_no_pip` has been set to `true`. This I will fix today.

**Comment by haxorof** [comment]
Reopen this issue if you still have the same problem with the newest version of this role.

Cheers!

**Comment by haxorof** [comment]
There is a bug in the role where I by mistake use a format in get_url module which only is available in Ansible 2.7 or later.
